apps/internal/base/internal/storage/storage.go

```diff
@@ -94,10 +94,7 @@ func (m *Manager) Read(ctx context.Context, authParameters authority.AuthParams,
 		return TokenResponse{}, err
 	}
 
-	accessToken, err := m.readAccessToken(homeAccountID, metadata.Aliases, realm, clientID, scopes)
```

Comment:
This is problem 2. This helper returns an error when it misses. A miss isn't fatal and using an error to signal one requires the caller to inspect the error to determine whether it's safe to proceed. I could've used a sentinel error for misses but that seemed excessive because the helper really doesn't need to return an error--it only searches a map.